doc/tutorials/active_dendrite/nestml_active_dendrite_tutorial.ipynb

```diff
@@ -0,0 +1,1162 @@
+      "INFO:Dependency analysis...\n",
+      "INFO:Saving dependency graph plot to /tmp/ode_dependency_graph_lin_cc.dot\n",
+      "INFO:Saving dependency graph plot to /tmp/ode_dependency_graph_analytically_solvable.dot\n",
+      "INFO:Generating numerical solver for the following symbols: V_m, I_kernel3__X__I_3, I_kernel2__X__I_2__d, I_kernel2__X__I_2, I_kernel1__X__I_1\n",
+      "INFO:In ode-toolbox: returning outdict = \n",
+      "INFO:[\n",
+      "    {\n",
+      "        \"initial_values\": {\n",
+      "            \"I_kernel1__X__I_1\": \"1\",\n",
+      "            \"I_kernel2__X__I_2\": \"0\",\n",
+      "            \"I_kernel2__X__I_2__d\": \"e/tau_syn2\",\n",
+      "            \"I_kernel3__X__I_3\": \"1\",\n",
+      "            \"V_m\": \"0\"\n",
+      "        },\n",
+      "        \"parameters\": {\n",
+      "            \"C_m\": \"250.000000000000\",\n",
+      "            \"E_L\": \"0\",\n",
+      "            \"I_e\": \"0\",\n",
+      "            \"tau_m\": \"20.0000000000000\",\n",
+      "            \"tau_syn1\": \"10.0000000000000\",\n",
+      "            \"tau_syn2\": \"10.0000000000000\",\n",
+      "            \"tau_syn3\": \"10.0000000000000\"\n",
+      "        },\n",
+      "        \"solver\": \"numeric\",\n",
+      "        \"state_variables\": [\n",
+      "            \"V_m\",\n",
+      "            \"I_kernel3__X__I_3\",\n",
+      "            \"I_kernel1__X__I_1\",\n",
+      "            \"I_kernel2__X__I_2\",\n",
+      "            \"I_kernel2__X__I_2__d\"\n",
+      "        ],\n",
+      "        \"update_expressions\": {\n",
+      "            \"I_kernel1__X__I_1\": \"-I_kernel1__X__I_1/tau_syn1\",\n",
+      "            \"I_kernel2__X__I_2\": \"1.0*I_kernel2__X__I_2__d\",\n",
+      "            \"I_kernel2__X__I_2__d\": \"-(I_kernel2__X__I_2 + 2*I_kernel2__X__I_2__d*tau_syn2)/tau_syn2**2\",\n",
+      "            \"I_kernel3__X__I_3\": \"-I_kernel3__X__I_3/tau_syn3\",\n",
+      "            \"V_m\": \"(C_m*(E_L - V_m) + tau_m*(I_dend_sp + I_e + I_kernel1__X__I_1 + I_kernel2__X__I_2 + I_kernel3__X__I_3))/(C_m*tau_m)\"\n",
+      "        }\n",
+      "    }\n",
+      "]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[14,iaf_psc_exp_nonlineardendrite_nestml, INFO, [2:0;71:0]]: Start building symbol table!\n",
+      "[15,iaf_psc_exp_nonlineardendrite_nestml, INFO, [2:0;71:0]]: Successfully generated code for the neuron: 'iaf_psc_exp_nonlineardendrite_nestml' in: '/tmp/nestml-active-dend-target' !\n",
+      "[16,GLOBAL, INFO]: Successfully generated NEST module code in '/tmp/nestml-active-dend-target' !\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open(\"iaf_psc_exp_nonlineardendrite.nestml\", \"w\") as nestml_model_file:\n",
+    "    print(nestml_active_dend_model, file=nestml_model_file)\n",
+    "\n",
+    "to_nest(input_path=\"iaf_psc_exp_nonlineardendrite.nestml\",\n",
```

Comment:
I suggest disabling by default the logging INFO (shown in red). Especially, this is not necessary to show here: https://nestml-sandbox.readthedocs.io/en/latest/tutorials/active_dendrite/nestml_active_dendrite_tutorial.html

Reply:
Agreed. Default logging level has been reduced to WARNING.